tests/test_overdrive.py

```diff
@@ -565,6 +565,8 @@ def test_update_licensepool_provides_bibliographic_coverage(self):
         availability['id'] = identifier.identifier
         bibliographic['id'] = identifier.identifier
 
+        self.api.queue_response(200, content=availability)
+        self.api.queue_response(200, content=bibliographic)
```

Comment:
This test would read better if you queued these responses just before calling `update_licensepool` on line 608.